syntax = "proto3";

package cave_talk;

enum Say
{
    SAY_OOGA = 0;
    SAY_BOOGA = 1;
}

message OogaBooga {
    Say ooga_booga = 1;
}